src/routes/subscriptions/subscription.controller.spec.ts

```diff
@@ -49,6 +54,8 @@ describe('Subscription Controller tests', () => {
     const moduleFixture: TestingModule = await Test.createTestingModule({
       imports: [AppModule.register(testConfiguration), EmailControllerModule],
     })
+      .overrideModule(JWT_CONFIGURATION_MODULE)
```

Comment:
As this opts in to `features.email`, the override is needed as`AuthGuard` is now on the `EmailController`.